src/deadline/client/ui/widgets/shared_job_settings_tab.py

```diff
@@ -184,7 +184,8 @@ def _handle_queue_parameters_update(self, refresh_id, queue_parameters):
             # Apply the initial queue parameter values
             for parameter in queue_parameters:
                 if parameter["name"] in self.initial_shared_parameter_values:
-                    parameter["value"] = self.initial_shared_parameter_values[parameter["name"]]
+                    if parameter["default"] == "":
```

Comment:
I see what you're doing here, and the intent is great. But, it introduces an ambiguity between an intentional default "" and the lack of a default. It could be in another case we don't want the empty string to be special like here. We didn't think very deeply about "null" values for the open jd types. Can you think of other ways we might fix this up? If not what you've got here may be the pragmatic choice even with the inconsistency.